content/zh-tw/observability.md

```diff
@@ -0,0 +1,19 @@
+---
+title: 可觀測性
+status: Completed
+category: 概念
+tags: ["屬性", "", ""]
+---
+
+可觀察性是一個系統屬性，其定義了系統能夠產生可執行的見解的程度。
+它允許使用者從這些外部輸出去理解系統狀態並採取行動來修正。
+
+計算機系統通過觀察低級信號進行衡量，例如 CPU 時間、記憶體、磁碟空間，以及更高級和業務信號，包括 API 回應時間、錯誤、每秒交易次數等。
+這些可觀察的系統通過專業工具，即所謂的可觀察性工具，進行觀察（或監控）。這些工具的列表可以在 Cloud Native Landscape 的可觀察性部分 中查看。
```

Comment:
```suggestion
這些可觀察的系統通過專業工具，即所謂的可觀察性工具，進行觀察（或監控）。這些工具的列表可以在 [Cloud Native Landscape](https://landscape.cncf.io/card-mode?category=observability-and-analysis&grouping=category) 的可觀察性部分 中查看。
```

Reply:
```suggestion
這些可觀察的系統通過專業工具，即所謂的可觀察性工具，進行**觀察**（或監控）。
這些工具的列表可以在 [Cloud Native Landscape 的可觀察性部分](https://landscape.cncf.io/card-mode?category=observability-and-analysis&grouping=category) 中查看。
```

EN
```
These observable systems are **observed** (or monitored) through specialized tools, so-called observability tools. A list of these tools can be viewed in the [Cloud Native Landscape's observability section](https://landscape.cncf.io/card-mode?category=observability-and-analysis&grouping=category).
```

Hyperlinks must be consistent with the original text

Reply:
Fixed, I accidentally kept my original draft, so what's why I have the repeat content before.